src/game/server/entities/character.h

```diff
@@ -26,6 +26,8 @@ class CCharacter : public CEntity
 	MACRO_ALLOC_POOL_ID()
 
 public:
+	#define POSITION_HISTORY 50
```

Comment:
No. Just, no. Move that out of the class or make it a static member.

Reply:
;)